dist-test/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+export PROJ_ROOT=$(dirname $(dirname $(readlink -f $0)))
+pushd ${PROJ_ROOT}/dist-test >> /dev/null
+
+# Set up image name
+if [[ -z "${FAASM_CLI_IMAGE}" ]]; then
+    VERSION=$(cat ../VERSION)
+    export FAASM_CLI_IMAGE=faasm/cli:${VERSION}
+fi
+
+if [ "$1" == "local" ]; then
+    INNER_SHELL=${SHELL:-"/bin/bash"}
+
+    # Start everything in the background
+    docker-compose \
+        up \
+        --no-recreate \
+        -d \
+        master
```

Comment:
Do you think we could also add a parameter to set the number of workers? It may be useful to have an arbitrary number of workers.